### DISCUSSION TITLE
/exception/gailan.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#57] [Announcements]
# /exception/gailan.html

Java程序员进阶之路，小白的零基础Java教程，从入门到进阶，一文彻底搞懂Java异常处理，YYDS

https://tobebetterjavaer.com/exception/gailan.html

**Comment by ConfigCpuTimer** [comment]
笔试选择题中经常有 try、catch 和 finally 块中都存在 return 的情况，希望可以补充。
